demo/verify_demo.py

```
@@ -11,7 +12,14 @@ def run_demo():
     demo_dir = os.path.dirname(os.path.abspath(__file__))
     print(f"📂 Parsing Terraform in: {demo_dir}")
     
-    resources = parser.parse_directory(demo_dir)
+    try:
+        resources = parser.parse_directory(demo_dir)
+    except ParseError as e:
+        print(f"❌ DEMO FAILED: Terraform parse error(s):")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)